docs/getting-started/elements/introduction.md

```diff
@@ -0,0 +1,28 @@
+# Elements
+
+Elements provides a component for embedding API reference documentation into any existing CMS, website or React application. Here are some of the features it supports:
+
+- OpenAPI v2 (Swagger), v3.0, v3.1 and beyond
+- Interactive "try it" (API Console)
+- Beautiful display of complex schemas, requests and responses
+- Embeddable
+- Multiple layouts
+
+Check out the demo for a live example https://elements-demo.stoplight.io.
+
+> Elements can be used on your local machine and in a production environment completly for free, no strings attached.
```

Comment:
`completly` -> `completely`

Reply:
Done in bdd9ff54b1a2434dc52989bae4f17c194597fc8d